### PR TITLE
docs: include tsh install in connect your client tsh page

### PR DESCRIPTION
**Pull request by stevenGravy**
- have install tsh like in Get started page in connect your client `tsh`.  The install Teleport page it previously linked to is quite advanced for typical users.
- use CDN for downloads consistently

### CODE REVIEW

File: docs/pages/connect-your-client/tsh.mdx [hunk 1]
```diff
@@ -80,10 +80,12 @@ to call [`tsh login`](../reference/cli.mdx#tsh-login) in the beginning.
 
 ## Installing tsh
 
-Follow [these install instructions](../installation.mdx) to obtain the `tsh`
-binary. Ideally, install `tsh` of the same version as the version used in your
+Follow install instructions to obtain the `tsh` binary. Ideally, install
```
Comment (ptgott) [inline docs/pages/connect-your-client/tsh.mdx:83]: What does "Ideally" mean here? If this is intended to imply that sometimes it's okay for `tsh` to not have the same version as the one used in your Teleport cluster, I think we should be explicit about when that is.

File: docs/pages/connect-your-client/tsh.mdx [hunk 2]
```diff
@@ -878,5 +880,13 @@ exit
 end of session playback
 ```
 
+## Uninstalling tsh
+
+If you installed on MacOS with `brew` run `brew uninstall teleport`.
```
Comment (ptgott) [inline docs/pages/connect-your-client/tsh.mdx:885]: I'm not sure we want to be highlighting the presence of the unsupported Homebrew package.

File: docs/pages/connect-your-client/tsh.mdx (hunk 1 above)
Comment (stevenGravy) [reply docs/pages/connect-your-client/tsh.mdx]: Please see update.

File: docs/pages/connect-your-client/tsh.mdx [hunk 3]
```diff
@@ -80,10 +80,12 @@ to call [`tsh login`](../reference/cli.mdx#tsh-login) in the beginning.
 
 ## Installing tsh
 
-Follow [these install instructions](../installation.mdx) to obtain the `tsh`
-binary. Ideally, install `tsh` of the same version as the version used in your
+Follow install instructions to obtain the `tsh` binary. Install
```
Comment (ptgott) [inline docs/pages/connect-your-client/tsh.mdx:83]: Is there an easy way for end users to determine the major version of the Teleport cluster (e.g., `/webapi/ping`)? 

The instructions below also don't allow the user to select a version. Maybe turn the version within `install-tsh.mdx` into a parameter and, in this inclusion, use a `Var` component for the value of the parameter?

Comment (zmb3) [reply docs/pages/connect-your-client/tsh.mdx:83]: `tsh version` will show the version of the connected proxy, as long as `--client-only` isn't specified.

Comment (stevenGravy) [reply docs/pages/connect-your-client/tsh.mdx:83]: WE can add a expandable option to get their current version

Comment (stevenGravy) [reply docs/pages/connect-your-client/tsh.mdx:83]: Please see update @ptgott

File: docs/pages/connect-your-client/tsh.mdx [hunk 4]
```diff
@@ -80,9 +80,22 @@ to call [`tsh login`](../reference/cli.mdx#tsh-login) in the beginning.
...
+To find the version number you can get it from:
+- In the Web UI select user name in the upper right, then Help & Support.
+  The Teleport Version and other information will show.
+- Retrieve from the Teleport cluster using `curl` and `jq`. Replace the variable
+  with your proxy address (e.g. mytenant.teleport.sh for Teleport cloud).
+  ```code
+  $ curl https://<Var name="teleport.example.com" />/webapi/find | jq '.server_version'
+  "(=teleport.version=)"
+  ```
+</Details>
```
Comment (ptgott) [inline docs/pages/connect-your-client/tsh.mdx:96]:
```suggestion
Follow the instructions below to install the `tsh` binary. 

We recommend installing
`tsh` of the same major version as the version used in your
Teleport cluster. 

To find the version number, either:
- In the Web UI, select your username in the upper right, then click **Help & Support**. You will see the version of your Teleport cluster.
- Use `curl` and `jq`. Replace <Var name="teleport.example.com" />
  with your Proxy Service address (e.g. `mytenant.teleport.sh` for Teleport Enterprise Cloud):
  
  ```code
  $ curl https://<Var name="teleport.example.com" />/webapi/find | jq '.server_version'
  "(=teleport.version=)"
  ```
```
Various readability suggestions. I also think all users will benefit from the instructions within the `Details` box, so we can make it part of the main body text.

**Review by ptgott — APPROVED** [review]

File: docs/pages/includes/install-tsh.mdx
[the remark this answers is not in the document]
Comment (stevenGravy) [reply docs/pages/includes/install-tsh.mdx:1]: I think keep it simple for now. If they're on V12 they can select from the docs drop down and get that version.

[the remark this answers is not in the document]
Comment (stevenGravy) [reply docs/pages/includes/install-tsh.mdx:1]: did you still want changes @alexfornuto ?

**Review by alexfornuto — APPROVED** [review]

**Pull request merged**